### ISSUE TITLE
handleMenuClick() does not properly set attributes.

### ISSUE THREAD

**Issue by programnoir** [nitronova/StudyGameDev.com#4]
The current site uses the CSS display property to circumvent the responsibility of the scripts to properly collapse the `<details>` tags and configure the attributes of it and its parent `<divs>` accordingly.

**Comment by programnoir** [comment]
I'm going to wait until after I tackle the modular population issue before I start on this.

**Comment by programnoir** [comment]
I will temporarily assign this to will not fix until someone provides evidence that the way the site is working now is not accessible.
